src/ui-test/tests/command.palette.test.ts

```diff
@@ -54,4 +58,12 @@ describe('JBang commands execution through command palette', function () {
         await executeCommand(CAMEL_RUN_ACTION_LABEL);
         await waitUntilTerminalHasText(driver, TEST_ARRAY_RUN);
     });
+
+    it(`Execute command '${CAMEL_RUN_DEBUG_ACTION_LABEL}' in command palette`, async function () {
+        await executeCommand(CAMEL_RUN_DEBUG_ACTION_LABEL);
+        await waitUntilTerminalHasText(driver, TEST_ARRAY_RUN_DEBUG);
+        await disconnectDebugger(driver);
+        const activityBar = new ActivityBar();
+        await (await activityBar.getViewControl('Explorer')).openView();
```

Comment:
@djelinek Here it is already important or Debug view will be passed to next test (tested with/without)

Reply:
or you can close sideBar in general.. it is not necessary and each test should handle itself..

Reply:
@djelinek but it will brake next tests too :disappointed: :

Run with solution from [here](https://github.com/camel-tooling/camel-dap-client-vscode/blob/4edf7f0fc9aba6e778d6a40dd6e443b7fb423690/src/ui-test/tests/editor.actions.test.ts#L91) 

  JBang commands execution through command palette
    ✔ Execute command 'Run Camel Application with JBang' in command palette (12302ms)
    ✔ Execute command 'Run Camel Application with JBang and Debug' in command palette (15599ms)

  Camel file context menu test
    Camel Group items
      1) "before all" hook: Before setup for "Debug and Run menu item is available"

  Camel file editor test
    Camel Actions
      2) "before all" hook: Before setup for "Debug and Run action is available"
      3) "after all" hook: After cleanup for "Can execute 'Run Camel Application with JBang' action"

  Install test
    Marketplace presentation
      ✔ Has correct title

Reply:
=> I have added to "before all" 
`await (await new ActivityBar().getViewControl('Explorer') as ViewControl).openView();` 
As I warned previously

Reply:
BTW we can put that "before all" as some util as "default" stay for test where we need explorer with folder (except only install)

Reply:
^ [here](https://github.com/camel-tooling/camel-dap-client-vscode/pull/389/commits/1eac20a8a45c50cfdcae6125bb6295b0591b1dad#diff-0d36ac6cda0dcf10be862d40f72a4ff5517868653e4da383b013046a69c8ceceR74-R91)

Reply:
BTW @djelinek I have faced that: 

debugger view is opened only if run&debug was called through the command palette, otherwise debug is started but the view is still `Explorer` :thinking: is it predicted behavior?

Reply:
I dont think so.. when I was testing run&debug editor action button I was always switched to Run and Debug view

Reply:
@apupier can you provide us here some context please? is this candidate for bug? any ideas?

Reply:
I think that it depends what is done during the previous (here first) run/debug session. If it moved to Explorer before stopping the debug session, VS Code should try to get back to previous state.

Reply:
@unsortedhashsets please open JIRA for this beahviour to investigate more later